.github/workflows/build.yml

```diff
@@ -118,3 +118,20 @@ jobs:
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           RUNTIME_PARAM: ${{ matrix.runtime-param }}
         run: ./scripts/run_integration_tests.sh
+
+      - name: Send success metric
```

Comment:
Do we even need a success metric?

Would it be simpler to just send a "failures" metric, and expect the value of the metric to be 0?

That way, the metric can be called "failures" instead of "status". It would work kind of like the other errors metrics that we have, like `lambda.errors`

Reply:
Or alternatively, if we do want to count successes, maybe it should be a separate `.success` metric?

I think right now the metric is being conceived as a `GAUGE` type where we have a decimal value between 0 (all successes) and 1 (all failures). I think it would be more intuitive to have the metric be a `COUNT` type where we just send a data point each time a failure occurs. (We could optionally have another metric for successes).

https://docs.datadoghq.com/metrics/types/?tab=count#metric-types

Also, do we need to specify the metric type in the API call?

Reply:
I found it useful as a condition to recover for an alert in the monitor. Would it be possible to do that with one single metric/value?

Reply:
Talked about this on Zoom -- I got confused here between the behavior of `GAUGE` and `RATE`. This solution looks good.